src/Ledger/Foreign/LedgerTypes.agda

```diff
@@ -85,6 +85,10 @@ record PParams : Set where
         poolDeposit   : Coin
         Emax          : Epoch
         pv            : Pair ℕ ℕ
+        ccTermLimit   : ℕ
+        govExpiration : ℕ
```

Comment:
What does this expiration apply to? It's not very clear from the name.

Reply:
It's how many epochs a governance action has before it expires. The name is from the CIP and there'll be some explanation in the prose later.